data/state/opt-in/MT/revenue-distribution.tsv

```diff
@@ -1,30 +1,30 @@
-Non-Metalliferous Mineral Leases Rental Income	Royalty/Rent/Bonus	$10,556				$10,196.00																									$360.00																								
-Oil & Gas Surface Damages	Royalty/Rent/Bonus	$9,950				$9,177.16																																	$772.80																
-Oil & Gas Seismic Permits	Royalty/Rent/Bonus	$5,688				$5,688.00																																																	
-Metalliferous Mineral Lease Rental Income	Royalty/Rent/Bonus	$2,842				$2,842.08																																																	
-Metalliferous Mineral Lease Royalty Income	Royalty/Rent/Bonus	$1,029				$1,028.79																																																	
-Land Use Licenses Royalty Income	Royalty/Rent/Bonus	$1,000				$1,000.00																																																	
-Micaceous Mines License Tax	License Tax	$-	$-																																																				
-All	All	$446,883,245	$176,559,116	$12,276,757	$133,478,180	$34,725,031	$28,838,092	$11,773,273	$6,921,142	$4,732,701	$3,583,888	$3,369,568	$3,219,424	$3,157,313	$3,149,120	$2,505,508	$2,454,052	$2,129,440	$1,185,166	$976,019	$732,488	$547,924	$372,716	$372,716	$366,000	$363,360	$348,578	$272,106	$250,000	$150,000	$66,684	$60,168	$45,007	$32,738	$24,086	$18,609	$12,990	$8,840	$7,468	$5,659	$2,087	$1,913	$210	$42	$-	$-	$-	$-	$-	$-	$-	$-	$-	$-	$-   
+Year	Revenue Stream	Type	Total	General Fund	State Share	County Share / Government	Common Schools	Coal Trust Fund	State School Oil & Gas Distribution	Long Range Building	Oil and Natural Gas Conservation Account Tax	State Orphan Share Fund	State Natural Resources Projects	Montana University System	Public Land Trust	Shared Account	Board of Oil & Gas Conservation	State Natural Resources Operations	Coal Board	Hard Rock Debt Service	Natural Resources Operations	Park Acquisition Trust	Renewable Resource Debt Service	Hazardous Waste/CERCLA special revenue account	Environmental Quality Protection Fund	Groundwater Assessment Account	Cultural and Aesthetic Projects	Hard Rock Mining	CERCLA Debt Service	Coal & Uranium Program	Water Storage Special Revenue Account	Public Buildings	School of Mines	Pine Hills School	Montana State University Morrill	University of Montana	State Normal School	Agricultural Experiment Station	School for Deaf and Blind	Department of Transportation	Department of Natural Resources Conservation - Water Resources Division	Montana State University 2nd Grant	Veterans Home Income	Department of Fish, Wildlife, & Parks	Galen State Hospital	State Guarantee Fund	County Oil & Natural Gas Impact Fund	State School Oil & Natural Gas Impact Fund	Coal Bed Methane Protection	Montana Tech	State Reform School	Department of Public Health and Human Services	Cities/Towns	County Wide School Ret/Trans	Local Schools	Miscellaneous & Fire
+2014	Oil & Natural Gas Production Tax	Production Tax	$236,496,773	$109,606,216		$95,997,576			$11,773,273		$4,732,701	$3,583,888	$2,624,135	$3,219,424			$2,505,508	$2,454,052																												$-	$-	$-	$-							
```

Comment:
Do we want to make a note in the opt-in README about adding a column? Or are we getting this from a source where we can request a different format?